straxen/plugins/led_calibration.py

```diff
@@ -141,3 +142,80 @@ def get_area(records, led_window):
     Area['area']    = Area['area']/float(len(end_pos))
         
     return Area
+
+
+@export
+@strax.takes_config(
+    strax.Option('channel_map', track=False, type=immutabledict,
+                 help="immutabledict mapping subdetector to (min, max) "
+                      "channel number."),
+)
+class nVetoExtTimings(strax.Plugin):
+    """
+    Plugin which computes the time differnce from external trigger timing to hitlets_nv.
+    The nearlest `time` of `raw_records_nv` before `hitlets_nv` time are used as the
+    external trigger timings.
```

Comment:
Actually this is not true.

 `Plugin which computes the time differnce from external trigger timing to hitlets_nv.` is wrong. You compute the time difference between a hitlet and the pulse start it was recorded in. But since our digitizer are connected in a daisy chain scheme the pulse start does not correspond to the trigger start. In fact there can be an offset of several 10ish ns.

Strictly speaking this is not true either `The nearlest `time` of `raw_records_nv` before `hitlets_nv` time are used as the
    external trigger timings.` You are using the `time` of the recorded pulse. `raw_records_nv` also includes the higher fragments of a pulse which could be in principle closer to the hitlet.

Reply:
Thank you. I fixed the comments. I drop my opinion here.
I reported the issue of the DAQ on telecon, but it has not been fixed. In my general thought, it should be connected to one clock with the same length cable when we need synchronisation. In addition, the trigger wasn't took "AND" on the dizitizer clock, it will make +/-8 ns or +/-16 ns offset in each channels. Moreover, it will occured among each mezzanines, actually, it has clearly been observed.
Without additional hardware module, CAEN digitizer has hardware delay offset thus I hope that the pulse start should correspond to the trigger start for future.
In some nVeto telecon, we were agreed to connect the trigger pulse to one available channel, but it hasn't. 

Thus, this is a one possible solution now.

Reply:
We can discuss this in todays telecon.